### PR TITLE
Update to Ukraine page

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Update to Ukraine page

### Trello card

https://trello.com/c/kL4o4etD/6158-ukrainian-visa-update

### Pre-election period restrictions

* [ ] This change is permitted within the constraints of the [pre-election period guidelines](https://www.gov.uk/government/publications/election-guidance-for-civil-servants/general-election-guidance-2024-guidance-for-civil-servants-html#publishing-content-online-)

### CODE REVIEW

**Comment by laurahermionetennant92** [comment]
@gemmadallmandfe got in a muddle and had to delete the first Ukraine branch!
Here is is again :)

File: app/views/content/non-uk-teachers/ukraine.md [hunk 1]
```diff
@@ -43,24 +43,18 @@ We can help if you’re interested in teaching children aged 5 to 18. If you'd l
...
-
-* the 2024 to 2025 academic year (which runs from 1 September 2024 to 31 August 2025) 
-
-[Learn more about the IRP](/non-uk-teachers/get-an-international-relocation-payment).
+If you are a teacher coming to England to teach languages or physics, you may be eligible for the IRP.[Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment).
 
 
 ### Visas giving you the right to train to teach or teach in England
 
 You already have the right to train or work as a teacher in England if you're in the UK on (one of the following):
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/ukraine.md:53]: Suggest removing the brackets around one of the following

File: app/views/content/non-uk-teachers/ukraine.md [hunk 2]
```diff
@@ -74,10 +68,10 @@ If you're already in the UK under a [different type of visa](https://www.gov.uk/
 
 ### Staying in the UK long-term
 
-If you're interested in working as a teacher and applying for [indefinite leave to remain in the UK](https://www.gov.uk/indefinite-leave-to-remain), you should consider a [skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).   
+If you're interested in working as a teacher and applying for [indefinite leave to remain in the UK](https://www.gov.uk/indefinite-leave-to-remain), you should consider a [Skilled Worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/ukraine.md:71]: @laurahermionetennant92 It looks like you are pointing to a page that was withdrawn in May 2023 so I think this link needs updating?

Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/ukraine.md:71]: Ouch well spotted

**Review by Sarah-DfE — APPROVED** [review]

**Pull request merged**